docker-compose.yml

```diff
@@ -9,4 +9,6 @@ services:
       - POSTGRES_USER=tiny_task
       - POSTGRES_PASSWORD=demo123
     volumes:
-      - ./postgres/data:/var/lib/postgresql/data
+      - pgdata:/var/lib/postgresql/data
```

Comment:
not part of task #32. what is the reason for this change and why volumes are used over mounts?